spec/factories/api_v1_posts.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :api_v1_post do
```

Comment:
```suggestion
  factory :post do
```

api_v1は不要かと！
ファイル名もspec/factories/posts.rbで良さそうです。
あとはpostのカラムのデータも指定した方がよさそうです！
rnameカラムなど

Reply:
@junjun491 
masterの方にはpostのファクトリーがありそうですね！
そしたらspec/factories/api_v1_posts.rbとspec/models/api_v1_post_spec.rbは実態はpostモデルになると思うので、不要かと思いましたがいかがでしょうか？？